### PR TITLE
Splitting the readme up into leapfrog and inc docs

### PR DESCRIPTION
**Pull request by antonym**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,326 +2,38 @@
...
-The next step is to execute the major upgrade script and follow the prompts:
-
-.. code-block:: shell
-
-    cd /opt/rpc-upgrades
-    scripts/ubuntu14-mitaka-to-newton.sh
+Incremental
+-----------
 
+Incremental upgrades are version to version upgrades that leverage the upstream run-upgrade.sh
```
Comment (rexredinger) [inline README.rst]: Maybe "Incremental upgrades are major to major version upgrades..."

File: incremental.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+==============================================
+
+Overview
+--------
+
+Incremental
+-----------
+
+Incremental upgrades are version to version upgrades that leverage the upstream run-upgrade.sh
+script provided in each release of Openstack Ansible.  They are used for releases for releases
```
Comment (rexredinger) [inline incremental.rst]: Same suggestions as above.

Comment (antonym) [reply incremental.rst]: Thanks, made those updates.

**Comment by antonym** [comment]
recheck_all

**Pull request merged**